packages/assets-controllers/src/RatesController/types.ts

```diff
@@ -0,0 +1,115 @@
+import type {
+  RestrictedControllerMessenger,
+  ControllerGetStateAction,
+  ControllerStateChangeEvent,
+} from '@metamask/base-controller';
+
+import type { fetchMultiExchangeRate as defaultFetchExchangeRate } from '../crypto-compare-service';
+import type { name as ratesControllerName } from './RatesController';
+
+/**
+ * Represents the conversion rates from one currency to others.
+ * Each key is a string representing the cryptocurrency code (e.g., "BTC", "SOL"),
+ * and its value is either a number representing the conversion rate to that currency,
+ * or `null` if the conversion rate is not available.
+ */
+export type Rate = Record<string, number | null>;
+
+/**
+ * Represents the conversion rates for multiple cryptocurrencies.
+ * Each key is a string representing the cryptocurrency symbol (e.g., "BTC", "SOL"),
+ * and its value is a `Rate` object containing conversion rates from that cryptocurrency
+ * to various fiat currencies or other cryptocurrencies.
+ */
+export type ConversionRates = Record<string, Rate>;
+
+/**
+ * Represents the state structure for the RatesController.
+ */
+export type RatesControllerState = {
+  /**
+   * The base currency for conversion rates.
+   */
+  currency: string;
+  /**
+   * The conversion rates for multiple cryptocurrencies.
+   */
+  rates: ConversionRates;
+  /**
+   * A list of supported cryptocurrency symbols.
+   */
+  fromCurrencies: string[];
+};
+
+/**
+ * Type definition for RatesController state change events.
+ */
+export type RatesControllerStateChangeEvent = ControllerStateChangeEvent<
+  typeof ratesControllerName,
+  RatesControllerState
+>;
+
+export type RatesControllerStartPollingEvent = {
```

Comment:
Same remarks than for the `startPolling` vs `pollingStart`

Reply:
Done in [265f6d1](https://github.com/MetaMask/core/pull/4242/commits/265f6d1e3d91b177eb851fd90d423c8648c1deb7)